src/zcl_abapgit_zip.clas.abap

```diff
@@ -132,9 +132,10 @@ CLASS ZCL_ABAPGIT_ZIP IMPLEMENTATION.
     ls_item-obj_type = ls_tadir-object.
     ls_item-obj_name = ls_tadir-obj_name.
 
-    lt_files = zcl_abapgit_objects=>serialize(
-      is_item     = ls_item
-      iv_language = sy-langu ).
+    zcl_abapgit_objects=>serialize(
```

Comment:
suggest having this RETURN a structure with files and flags instead of EXPORTING + IMPORTING + CHANGING

Reply:
changed to returning